modules/book_module.rb

```diff
@@ -0,0 +1,29 @@
+module BookModule
+  def capture_input(request_text)
+    print "#{request_text}: "
+    gets.chomp
+  end
+
+  def add_new_book_details
+    puts "\n-------------------------------"
+    puts "\nENTER BOOK DETAILS\n\n"
```

Comment:
There's no functionality for adding a new `author` on your app.

- Kindly create an option for adding a new author when creating a new book. Note that the publisher is not the author, but rather the company name that published the book.
- I should be able to see the list of all authors that were created when adding a new book. The authors' list is always empty.

Reply:
@KDlamini  Thank you for your feedback